CONTRIBUTING.md

```diff
@@ -122,6 +164,7 @@ changed and why. Follow these guidelines when writing one:
     Refs: http://eslint.org/docs/rules/space-in-parens.html
     Refs: https://github.com/nodejs/node/pull/3615
     ```
+    - It's it's not a fix, you should document the motivation for the change
```

Comment:
`It's it's` -> `If it's`

Punctuation at the end of the change.

Avoid personal pronoun.

> If it's not a fix, document the motivation for the change.